contracts/interfaces/IBondingMinter.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+interface IBondingMinter {
+    event StartBonding(uint startTimestamp, uint endTimestamp);
+    event Mint(address indexed tokenId, address indexed user);
```

Comment:
Why do we need both mint and bond issue events? Isn't it the same?

Reply:
In general it seems that what we really need probably is just the event MintData with all the data necessary, correct me if I'm wrong

Reply:
Resolving conversation by clearing the misunderstanding with comments